lib/stellar_core_commander/process.rb

```diff
@@ -605,17 +610,26 @@ def check_equal(kind, x, y)
 
     Contract Process => nil
     def check_equal_ledger_objects(other)
-      check_equal "account count", account_count, other.account_count
-      check_equal "trustline count", trustline_count, other.trustline_count
-      check_equal "offer count", offer_count, other.offer_count
+      if is_sqlite
+        $stderr.puts "#{idname}: check_equal_ledger_objects is not supported with SQLite DB option."
```

Comment:
same thing here: this is a silent failure (nobody reads logs), we should raise instead (and not call `check_integrity_against` in recipes that use sqlite)